dlp/inspect_content_test.py

```diff
@@ -439,7 +451,13 @@ def test_inspect_bigquery(bigquery_project, topic_id, subscription_id, capsys):
         topic_id,
         subscription_id,
         ["FIRST_NAME", "EMAIL_ADDRESS", "PHONE_NUMBER"],
-    )
+        timeout=1)
 
     out, _ = capsys.readouterr()
-    assert "Info type: FIRST_NAME" in out
+    assert "Inspection operation started" in out
+    # Cancel the operation
```

Comment:
Can we either do this as part of the fixture, or use `try` and `finally` to verify it happens if an exception is raised above?

Reply:
done with try-finally (with the next commit)